initial_pose: 
  x: 0
  y: 0
  z: 0
  R_deg: 0
  P_deg: 0
  Y_deg: 0